### ISSUE TITLE
Get server-error if I use "set colour sensor" with NXT

### ISSUE THREAD

**Issue by Mathias573** [OpenRoberta/robertalab#514]
Hi,

if I use the function "Setze Farbsensor" (set colour sensor?) in my program and then start the simulation or use the start-button to transfer the program to my NXT then I get an server-error and my whole program is deleted.

![setze_farbsensor](https://cloud.githubusercontent.com/assets/29167259/26756951/07a96c08-48ae-11e7-908a-cd9ba0113511.PNG)

![fehlermeldung](https://cloud.githubusercontent.com/assets/29167259/26756953/1c13cc06-48ae-11e7-90dd-d7d0886a4d46.PNG)

**Comment by Mathias573** [comment]
If I change the language then I get the the images in english :-)

![set_colour_sensor](https://cloud.githubusercontent.com/assets/29167259/26757055/f94d66c0-48b0-11e7-8b9f-e0dacdcd57ec.PNG)
![server-error](https://cloud.githubusercontent.com/assets/29167259/26757058/fff8f7e6-48b0-11e7-85c2-25ab926d45b9.PNG)

**Comment by Mathias573** [comment]
The same error if I read timestamp:
![error_time](https://cloud.githubusercontent.com/assets/29167259/26759688/bc63891a-4905-11e7-9b56-81c872d994b7.PNG)

If I build the program for EV3 there is no problem.

**Comment by Mathias573** [comment]
I made an other program:
![get_rotation_a](https://cloud.githubusercontent.com/assets/29167259/26797243/7d3163e8-4a2c-11e7-853d-e3ba7d7c8744.PNG)
I used this NXT-configuration:
![nxt_configuration](https://cloud.githubusercontent.com/assets/29167259/26797263/91a6aaea-4a2c-11e7-8f2a-68994cda90c5.PNG)
In simulation everything works fine.
If I try to transfer the program to NXT-robot, I get the "server-error". 
When I change the encoder from A to B or put a motor on port A in configuration then I can transfer the program to robot without errors.

But I've seen you commited a fix. So I think you found the cause for this behaviour and in future I will look for the right configuration of my NXT-robot. :-)
